model_analyzer/record/metrics_manager.py

```diff
@@ -367,6 +367,7 @@ def _create_model_variant(
                         self._output_model_repo_path, variant_name
                     )
             except FileExistsError:
+                # Ignore if the file already exists
```

Comment:
Unnecessary comment?

Reply:
This comment is intended to resolve https://github.com/triton-inference-server/model_analyzer/security/code-scanning/6